lib/ex_aws/operation/s3.ex

```diff
@@ -24,7 +24,6 @@ defmodule ExAws.Operation.S3 do
 
       url =
         operation
-        |> add_bucket_to_path
```

Comment:
Doesn't this break non-virtual-host configus, such that the bucket will never be added to the path now? IOW, this looks like it would work *only* with virtual host setups?